### DISCUSSION TITLE
Recommended Video Capture Hardware as June 2024?

### DISCUSSION THREAD

**Discussion by DastardlyDino** [awawa-dev/HyperHDR#884] [Show and tell]
I'm getting ready to buy the necessary hardware to put together my HyperHDR set up. Before making my purchases, I wanted to check in to see if the video capture hardware used in the guide is still the recommended hardware. The guide, which was last updated February 12, 2023 recommends either MS2109 or MS2130 for video grabbing and FeinTech VSP01201 or Ezcoo EZ-SP12H2 for video splitting. Is this still recommended?

**Comment by DastardlyDino** [comment]
For more context, I am currently running HyperHDR on Unraid and capturing the video info using Android Hyperon Grabber. Hyper HDR on Unraid then wireless transmit light info to WLED running on a dig-quad. It's working fine but I would like to be able to a bit more with my setup.

1. I would like to be able to capture video data from all input sources like my Nintendo switch. Hopefully I can still maintain easy source switching. 
2. I would like to be able to use HyperHDR with applications like Netflix & HBO on my Nvidia Shield.
3. I would like more accurate color output.

By moving to a dedicated raspberry pi setup with a video splitter and capture device I believe I'll be able to accomplish all 3 of these goals. My TV is a Samsung S95b.

**Comment by awawa-dev** [comment]
Hi
Avoid Feintech if you don't want to have a problem with DV (LLDV) later. ms2130 is still a very good choice if your device supports USB3.0, but you may need to flash the firmware (old or experimental) as there is a mess on the market at the moment. Ezcoo EZ-SP12H2 is still recommended,  some users have tested this matrix https://www.amazon.com/Matrix-4K7-1-Switch-Extractor-l-b-y/dp/B0B9GNNPB2 and are satisfied, but I do not have this model to confirm it. Probably the solution for your case may be one of Ezcoo matrix model.

**Reply by DastardlyDino** [reply]
Thank you for your recommendations

**Comment by Stefan136** [comment]
Hello,
any experience with FeinTech SP210, or is this Splitter not recommended as well?

**Reply by awawa-dev** [reply]
No, the main issue is how other Feintech models handle Dolby Vision (especially LLDV) compared to Ezcoo. If you have possibility to order and return the product (for ex. Amazon) then you probably will have to test it yourself. Other users in similar situation would appreciate the report from such testing.
